superset-frontend/src/components/Form/LabeledErrorBoundInput.tsx

```diff
@@ -0,0 +1,95 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import { Input } from 'antd';
+import { styled, css, SupersetTheme } from '@superset-ui/core';
+import FormItem from './FormItem';
+import FormLabel from './FormLabel';
+
+export interface LabeledErrorBoundInputProps {
+  label?: string;
+  name: string;
+  validationMethods:
+    | { onBlur: (value: any) => void }
+    | { onChange: (value: any) => void };
+  errorMessage: string | null;
+  helpText?: string;
+  value: string | number | readonly string[] | undefined;
+  required?: boolean;
+  placeholder?: string;
+  autoComplete?: string;
+  type?: string;
```

Comment:
I don't think this is being used below. You can remove it, and let it pass through in the props spread instead.

Reply:
we can replace it with a catchall: `[x: string]: any`

Reply:
Much cleaner! Fixed in [`this commit`](https://github.com/apache/superset/pull/14530/commits/f476e27f08885c461b448c6b6042f5d149ec1935)